### ISSUE TITLE
Wunderbar invisible at start

### ISSUE THREAD

**Issue by Krekot** [Toxicom/toxiui#8]
### Description

Wunderbar is invisible at login or reload need to change anything to make it visible again like texture, bg color etc. Tested only with ElvUi, Windtools and ToxiUi

### Screenshots

![image](https://user-images.githubusercontent.com/580730/198874948-7a0a41e4-a5e7-4642-9d00-777a9449f596.png)

### To reproduce

1.

### Acceptance criteria

-   [ ]

**Comment by Krekot** [comment]
Ok I suppose it is problem with 
![image](https://user-images.githubusercontent.com/580730/198875199-5c2f57db-7000-40ac-bc2e-cc5e3623e127.png)

At max lvl it is invisible same with clock. Trying to change module give this error:
```lua
7x ...lvUI/Core/Modules/UnitFrames/Elements/CustomText.lua:19: attempt to index local 'frameDB' (a nil value)
[string "@Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/CustomText.lua"]:19: in function `Configure_CustomTexts'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Misc/Tags.lua"]:33: in function `_TagsUpdate'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Misc/Tags.lua"]:42: in function `callback'
[string "@Interface/AddOns/ElvUI_ToxiUI/Core/Functions/Event.lua"]:312: in function `ContinueAfter'
[string "@Interface/AddOns/ElvUI_ToxiUI/Core/Functions/Event.lua"]:351: in function `ContinueAfterElvUIUpdate'
[string "@Interface/AddOns/ElvUI_ToxiUI/Modules/Misc/Tags.lua"]:41: in function <Interface/AddOns/ElvUI_ToxiUI/Modules/Misc/Tags.lua:40>

Locals:
self = <table> {
 Update_AssistHeader = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Groups/Assist.lua:42
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/Ace3/AceTimer-3.0/AceTimer-3.0.lua:147
 RaidRoleUpdate = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/RaidRoleIcons.lua:52
 SetSmartPosition = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Auras.lua:291
 Construct_PowerBar = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Power.lua:14
 arena1 = ElvUF_Arena1 {
 }
 RegisterForClicks = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/UnitFrames.lua:1544
 Configure_AllAuras = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Auras.lua:275
 Construct_Buffs = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Auras.lua:58
 Construct_PowerPrediction = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/PowerPrediction.lua:34
 ScheduleTimer = <function> defined @Interface/AddOns/Ace3/AceTimer-3.0/AceTimer-3.0.lua:94
 UpdateNameSettings = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Name.lua:14
 ADDON_LOADED = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/UnitFrames.lua:1337
 FrameGlow_CheckMouseover = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/FrameGlow.lua:299
 Configure_PVPText = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/PVPText.lua:8
 MatchGrowthX = <table> {
 }
 db = <table> {
 }
 Update_TankFrames = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Groups/Tank.lua:71
 CreateRaisedText = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/UnitFrames.lua:242
 SetTexture_HealComm = <function> defined @Interface/AddOns/Ace3/AceHook-3.0/AceHook-3.0.lua:100
 Hook = <function> defined @Interface/AddOns/Ace3/AceHook-3.0/AceHook-3.0.lua:273
 SetStatusBarBackdropPoints = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/UnitFrames.lua:1443
 GetTalentTicks = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/CastBar.lua:360
 ClassPower_SetBarColor = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/ClassBars.lua:50
 BuffIndicator_PostUpdateIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/BuffIndicator.lua:69
 FrameGlow_SetGlowColor = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/FrameGlow.lua:177
 Construct_BossFrames = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Groups/Boss.lua:11
 GetAuraRow = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Auras.lua:94
 Construct_RaidDebuffs = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/RaidDebuffs.lua:8
 PostUpdate_PhaseIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/PhaseIndicator.lua:4
 assist = ElvUF_Assist {
 }
 AuraUnchanged = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/Auras.lua:562
 Construct_RaidpetFrames = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Groups/RaidPets.lua:5
 groupPrototype = <table> {
 }
 Construct_InfoPanel = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/InfoPanel.lua:6
 Configure_HealthBar = <function> defined =[C]:-1
 PostUpdateBar_AuraBars = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/AuraBars.lua:192
 GetName = <function> defined @Interface/AddOns/Ace3/AceAddon-3.0/AceAddon-3.0.lua:275
 PostColorAdditionalPower = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/ClassBars.lua:510
 classMaxResourceBar = <table> {
 }
 name = "ElvUI_UnitFrames"
 SetAlpha_HealComm = <function> defined @Interface/AddOns/ElvUI/Core/Modules/UnitFrames/Elements/HealPrediction.lua:12
 boss7 = ElvUF_Boss7 {
 }
 Disable = <function> d
```

At low lvl character it is visible

**Comment by Krekot** [comment]
When I removed Databar module from Wunderbar at my low lvl character everything is working. Could not remove it at max lvl character bcs was getting that error.

**Comment by bagelbits** [comment]
I think this is still broken. But should be fixed with #13

**Comment by Toxicom** [comment]
Is this still happening with beta 9? @Krekot
